### DISCUSSION TITLE
Which bird is believed to have the biggest potential vocabulary?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#144] [Galaxy Trivia] [answered]
**Hint:** 🦜

**Comment by Tynab** [comment]


**Reply by ParanoidUser** [reply]
The [Common myna](https://en.wikipedia.org/wiki/Common_myna) can "talk," but not as well as its African cousin. Can you guess?

**Reply by torhaa1** [reply]
African Grey Parrot is also known for large vocabulary and to mimic human speech

**Comment by torhaa1** [comment, marked as answer]
The African Grey Parrot is also known for large vocabulary and to mimic human speech
